test/functionalTest/cases/1316_create_subscription/post_subs_expr_q_lesser_or_equal_than.test

```diff
@@ -0,0 +1,216 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+POST Subscriptions V2 q expression (lesser or equal than)
```

Comment:
I'm using the same wording that the one in http://telefonicaid.github.io/fiware-orion/api/v2/ for the Simple Query Language section.

If this is wrong, it should be changed also in that document (a bit surprising anyway, given that the .apib has been reviewed many times by many people and such typo should has been quickly spotted in that process...)